ui/app/AppLayouts/Onboarding2/pages/KeycardCreatePinPage.qml

```diff
@@ -7,15 +7,23 @@ import StatusQ.Components 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Controls.Validators 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Backpressure 0.1
 
 import AppLayouts.Onboarding2.controls 1.0
+import AppLayouts.Onboarding.enums 1.0
 
 import utils 1.0
 
 KeycardBasePage {
     id: root
 
+    required property int keycardPinInfoPageDelay
+    required property int pinSettingState
+    required property int authorizationState
+
     signal keycardPinCreated(string pin)
+    signal keycardPinSuccessfullySet()
```

Comment:
The caller fully knows the state of setting pin because provides the `pinSettingState`. This signal seems to be not necessary at all.